sample/README_mysql.md

```diff
@@ -0,0 +1,143 @@
+# Mysql Sample
+
+This sample is for experimenting with the Airbyte Module (ABM) mysql connetor.
+
+The server is configured to read and write a mysql table.
+
+It uses Kuberentes for the mysql chart deployment only while the Airbyte Module (ABM) runs locally without Kubernetes.
```

Comment:
I suggest: `It uses Kuberentes only for the mysql chart deployment, while the Airbyte Module (ABM) runs locally without Kubernetes.`

Reply:
Fixed. Thanks